modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,177 @@
+
+*  Before this update, the `disable-ha` flag was not correctly passed to the Pipelines controller, and the high availability (HA) functionality was never enabled. With this update, the HA functionality is enabled if you set the `disable-ha` flag to `false`.
+
+* Before this update, the `skopeo-copy` cluster task would fail when attempting to copy images mentioned in  ConfigMap data. With this update, the `skopeo-copy` cluster task completes properly.
+
+* With this update, a pipeline run automatically generated by the `tkn pac generate –language=java` command has correct annotations and parameter names.
+
+* Before this update, only a user with the administrative permissions could successfully run the `tkn pac create repository` command. With this update, any authorized user can run the `tkn pac create repository` command.
+
+* Before this update, the `/test <run-name>` and `/retest <run-name>` user comments, which specified a particular pipeline, did not trigger pipeline runs as expected. With this update, these comments trigger pipeline runs successfully.
+
+* Before this update, if there were multiple pipeline runs in the `.tekton` folder with the `generateName` field and not the `Name` field, the pipeline runs failed. This update fixes the issue.
+
+* Before this update, in {pac} when using GitLab, a pipeline run was triggered by any event in a merge request, including adding labels and setting status. With this update, the pipeline run is triggered only when there is an open, reopen, or push event. A comment containing the status of the checks is now posted on the merge request.
+
+* Before this update, while a pipeline run was waiting for approval, the status of the check could be displayed as `skipped`. With this update, the correct `pending approval` status is displayed.
+
+* Before this update, the bundles resolver sometimes set the type to `Task` when attempting to retrieve a pipeline, leading to errors in retrieval. With this update, the resolver uses the correct type to retrieve a pipeline.
+
+* This update fixes an error in processing the Common Expression Language (CEL) NOT operator when querying {tekton-results}.
+
+* This update fixed a `404` error response that was produced in the {tekton-results} API when a record LIST operation was requested and the specified result was `-`.
+
+* Before this update, in an EventListener object, the `status.address.url` field was always set to the default port. With this update, the `status.address.url` field is set to match the port specified in the `spec.resources.kubernetesresource.serviceport` parameter.
+
+* Before this update, if the GitHub API provided a paginated response, {pac} used only the first page of the response. With this update, all paginated responses are processed fully.
```

Comment:
We need to add one more point here for chains fix 

Before this update, Tekton chains controller was crashing when the host address of KMS Hashicorp Vault was misconfigured or chains was unable to connect to it , now with this update, it will log the correct error and will not panic in these conditions